src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -62,6 +62,7 @@
 import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
 import org.jenkinsci.plugins.workflow.job.WorkflowJob;
 import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.Assume;
```

Comment:
Running reliably on Windows in my tests without the Assume.  Let's check it on ci.jenkins.
```suggestion
```